README.md

```diff
@@ -1,6 +1,16 @@
-# example-npm-publish
+# Pusher Chatkit Gifted Chat
 
-An example repository for publishing to npm via CI
+A helpful way to consume [Pusher Chatkit](https://pusher.com/chatkit) while using `react-native-gifted-chat` or `react-web-gifted-chat`
+
+## Installation
+
+`npm install pusher-chatkit-gifted-chat`
+
+or
+
+`yarn add pusher-chatkit-gifted-chat`
+
+The project expects you to have `react` and `prop-types` installed but has no other dependencies.
```

Comment:
Why do we need prop-types? It shouldn't be necessary with TypeScript and I can't find any references in code.

Reply:
Ok I'll drop it